qutebrowser/misc/crashsignal.py

```diff
@@ -430,6 +432,15 @@ def interrupt_really_forcefully(self, signum, _frame):
         print("WHY ARE YOU DOING THIS TO ME? :(")
         sys.exit(128 + signum)
 
+    def reload_config(self, _signum, _frame):
+        """Reload the config."""
+        log.signals.info("SIGHUP received, reloading config.")
+        config_commands = objreg.get('config-commands', from_command=True)
+        try:
+            config_commands.config_source()
+        except cmdutils.CommandError as e:
```

Comment:
I feel like we should do a `message.error()` in this except block too so that the user is aware. Or do you think these signals can sometimes come from sources outside a user's control?